test_base_import_pdf_by_template/demo/base_import_pdf_template.xml

```diff
@@ -113,140 +113,4 @@
         <field name="value_type">variable</field>
         <field name="log_distinct_value" eval="True" />
     </record>
-    <!-- Account invoice Template Data !-->
```

Comment:
Why removing all of this?

Reply:
Previously the account template and the test were in `test_base_import_pdf_by_template`. Now, with the new module (which already depends on `account`) I have moved that to this new module and used that template to create a test of the process of creating an invoice from the PDF that is processed when receiving an email.

Reply:
But then, do we need the `test_*` module?

Reply:
For now, it is still necessary for a purchase order template (for example).

Reply:
OK